simplified-app-shared/src/main/java/org/nypl/simplified/app/catalog/CatalogFeedActivity.java

```diff
@@ -748,14 +733,32 @@ public Boolean onFeedSearchLocal(
             return NullCheck.notNull(Boolean.TRUE);
           }
         });
+
+    } else {
+      CatalogFeedActivity.LOG.debug("Feed has no search opts.");
     }
 
     if (search_ok) {
+      this.search_view.setSubmitButtonEnabled(true);
+      this.search_view.setIconifiedByDefault(false);
+      search_item.setShowAsActionFlags(MenuItem.SHOW_AS_ACTION_ALWAYS);
+      search_item.expandActionView();
+
+      this.search_view.setQueryHint("Search " + this.feed.getFeedTitle());
+      if (args.getTitle().startsWith("Search")) {
```

Comment:
I think there is probably a better way to determine if we have search than examining the title. We should be deciding this based on a successful parse of an OpenSearch document.

Reply:
There probably is, but let's explore that in a future branch.  For now, this is pre-existing code that I just moved around a bit.